### PR TITLE
Enhancements to Azure Adapter, Sample docs, and Azure artifacts 

### PR DESCRIPTION
**Pull request by brunoborges**
Use the latest versions of the Azure artifacts on Maven Central.
Move documentation about sample to its own README.
Remove GET support from function.

### CODE REVIEW

File: spring-cloud-function-samples/function-sample-azure/pom.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
 		<dependency>
 			<groupId>com.microsoft.azure</groupId>
 			<artifactId>azure-functions-java-core</artifactId>
-			<version>1.0.0-beta-2</version>
+			<version>1.0.0-beta-3</version>
```
Comment (dsyer) [inline spring-cloud-function-samples/function-sample-azure/pom.xml]: Probably this ought to go in the adapter itself right? There is no need for samples (user code) to know this version id?

**Comment by brunoborges** [comment]
Hey @dsyer , can you be more specific with your comment here? https://github.com/spring-cloud/spring-cloud-function/pull/178#pullrequestreview-125582588

File: spring-cloud-function-samples/function-sample-azure/pom.xml (hunk 1 above)
Comment (brunoborges) [reply spring-cloud-function-samples/function-sample-azure/pom.xml]: Good point.

**Comment by dsyer** [comment]
I just meant that the PR has conflicts (my fault, sorry). If you want to fix them, you probably have to manually move the asciidoc changes.

**Comment by dsyer** [comment]
I didn't like your merge so I tried to rebase (see 37d0d95). If that looks OK to you then I think we have all the changes from this PR.

**Comment by brunoborges** [comment]
@dsyer sure. Just let me know what you need.

**Pull request closed without merging**